client/src/game/movePlayer.js

```diff
@@ -0,0 +1,17 @@
+export default function movePlayer(event, player) {
```

Comment:
Hey. I tried to solve this by handling only the keyCode instead of the whole event. But `.keycode` seams to be deprecated in JavaScript ([`.which` as well](https://css-tricks.com/snippets/javascript/javascript-keycodes/)). I could solve this by using `.key` oder `.keyIdentifier` but they don't work in every browser. [Can I use `.keyCode`](https://caniuse.com/?search=keyCode), oder should I go save and use something else, which would lead to a code like this:
```js
if (event.key) {
} else if (event.keyIdentifier) {
} else if (event.keyCode) {
}
```
like it is presented on stackOverflow: https://stackoverflow.com/questions/35394937/keyboardevent-keycode-deprecated-what-does-this-mean-in-practice

seams a bit much for just one deprecated value.

Reply:
Good to know. Looks like `code` is a good choice:
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code
>You should avoid using this if possible; it's been deprecated for some time. Instead, you should use KeyboardEvent.code, if it's implemented. Unfortunately, some browsers still don't have it, so you'll have to be careful to make sure you use one which is supported on all target browsers.

It's not supported by IE11, but who cares 😀.

Reply:
okay. Thanks for your help. I will try it out ;)